tests/fixtures/problem/openapi.yaml

```diff
@@ -79,6 +79,13 @@ paths:
       responses:
         '200':
           description: Problem exception
+  /value_error:
+    get:
+      description: Simply raise a ValueError
+      operationId: fakeapi.hello.throw_value_error
+      responses:
+        '200':
```

Comment:
shouldn't this be 599?

Reply:
yes.. absolutely! I've fixed this.

Reply:
I think what is happening here is that GitHub does not automatically track updates to closed PRs. If you look at the commit in my own fork then you'll see that the line is changed: https://github.com/samuelgrigolato/connexion/blob/fix_returning_problem_in_error_handlers/tests/fixtures/problem/swagger.yaml#L112

Reply:
Just for the sake of completion, this is the reference to the same line in the `openapi.yaml` file (which you originally pointed out in your review): https://github.com/samuelgrigolato/connexion/blob/fix_returning_problem_in_error_handlers/tests/fixtures/problem/openapi.yaml#L82